pkg/node/local_node_store.go

```diff
@@ -18,6 +18,30 @@ type LocalNode struct {
 	// OptOutNodeEncryption will make the local node opt-out of node-to-node
 	// encryption
 	OptOutNodeEncryption bool
+	// hostEndpointID is the endpoint ID allocated for the host
+	hostEndpointID uint64
+	// ingressEndpointID is the endpoint ID allocated for the ingress endpoint on this node
+	ingressEndpointID uint64
+}
+
+// GetEndpointID returns the ID of the host endpoint for the LocalNode
+func (n *LocalNode) GetEndpointID() uint64 {
```

Comment:
What value do these getters and setters provide over exporting `HostEndpointID`?

Reply:
Point taken, except for being moot for now (see below).